roles/openshift_logging_kibana/tasks/main.yaml

```diff
@@ -71,8 +71,7 @@
   copy:
     content: "{{ 32 | lib_utils_oo_random_word }}"
```

Comment:
I am not sure this will work.  `lib_utils_oo_random_word` generates a value that oauth-proxy will interpret to be a base64 encoded string.  The decoded bytes will not meet the length requires for AES-256.  I believe you want a base64 string of length 43 so that it decodes in 32 bytes for an AES 256 bit key.

Reply:
@enj that does work for me though.
The string is saved into the file on filesystem, then secret is constructed from the file, then the secret is mounted in oauth-proxy. 
is there a filter to build base64 string?

Reply:
```
{{ encoded | b64decode }}
{{ decoded | b64encode }}
```
https://docs.ansible.com/ansible/2.5/user_guide/playbooks_filters.html